sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -32,6 +32,12 @@ public class DigitalTwinsClient
         /// </summary>
         /// <param name='endpoint'>The Azure digital twins service instance URI to connect to.</param>
         /// <param name="credential">The <see cref="TokenCredential"/> implementation which will be used to request for the authentication token.</param>
+        /// <seealso cref="DigitalTwinsClient(Uri, TokenCredential)">
+        /// This other constructor provides an opportunity to override default behavior, including specifying API version,
+        /// overriding <see href="https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/samples/Pipeline.md">transport</see>,
```

Comment:
I'm told I can use <see href="" /> for web links, but I don't know if I can enclose the link name in that tag. So I'll submit this and see how the web docs turn out.